data/quarg/quarg missions.txt

```diff
@@ -259,3 +259,47 @@ mission "Ask Quarg About Coalition Early"
 			label end
 			`	Before you can ask it anything else, the Quarg bids you farewell, and walks away.`
 				decline
+
+
+
+mission "Quarg Pug Arfecta Warning"
+	minor
+	landing
+	to offer
+		has "First Contact: Quarg: offered"
+		has "flagship model: Pug Arfecta"
+	source
+		attributes "quarg"
+		attributes "ringworld"
+	on offer
+		conversation
+			`As soon as you land your ship a crowd of visibly irritated Quarg start to gather, and as you go to disembark you find a rather imposing Quarg blocks your exit while staring piercingly at your ship.`
```

Comment:
```suggestion
			`As soon as you land your ship a crowd of visibly irritated Quarg start to gather, and as you go to disembark you find a rather imposing Quarg blocks your exit while staring piercingly at you.`
```
"Piercingly" strikes me as a strange word to describe the Quarg's gaze with respect to your ship because it begs the question, "what exactly are they piercing?" (The ship's hull?) I typically think of "piercing" used to describe how one person looks at another (i.e. "staring piercingly through you"), such that they are piercing obfuscations or lies or feigned ignorance, all of which the imposing Quarg demonstrates later in the conversation.

Reply:
Makes sense will update myself as this line now has a conflict.

Reply:
Change made